admin/controllers/users/views/edit/view.php

```diff
@@ -104,7 +104,9 @@ function in_group ($group_id, $edit_user) {
 	echo "<script>new SlimSelect({ select: '#usertags' });</script>"; 
 	?>
 
-	<?php Hook::execute_hook_actions('display_user_fields_form',$edit_user); ?>
```

Comment:
leave this hook here for back compat and other things that use it. probably put it after